src/pages/index.js

```diff
@@ -5,57 +5,175 @@ import FormValidator from "../components/FormValidator.js";
+export const cardAddModal = document.querySelector("#card-add-modal");
+//export const cardsList = document.querySelector(".cards__list");
+
+// Buttons and other DOM nodes
+const closeButtons = document.querySelectorAll(".modal__close");
+const profileEditBtn = document.querySelector("#profile-edit-button");
+const cardAddNewBtn = document.querySelector("#card-add-button");
+const editAvatarButton = document.querySelector("#edit-avatar-icon");
+
+// Form data
+
+export const avatarUrlInput = document.querySelector("#avatar-input-url");
+export const profileInputTitle = document.querySelector("#profile-input-title");
+export const profileInputDescription = document.querySelector(
+  "#profile-input-description"
+);
+
+export const addCardTitleInput = document.querySelector("#card-input-title");
+export const addCardImageInput = document.querySelector("#card-input-url");
+
+// API ------
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d2cb5c42-088c-4f10-972a-3d1426bf2382",
+    "Content-Type": "application/json",
+  },
+});
 
 // Section -----
 
 const section = new Section(
   {
-    items: initialCards,
-    renderer: (cardData) => {
-      const cardElement = createCard(cardData);
-      section.addItem(cardElement);
+    // items: initialCards,
+    renderer: (item) => {
+      const card = createCard(item);
+      section.addItem(card);
     },
   },
   ".cards__list"
 );
 
-section.renderItems();
+// INITIAL CARDS ------
 
-function createCard(item) {
-  console.log(item);
-  const cardElement = new Card(item, "#card-template", handleImageClick);
-  return cardElement.getView();
-}
+api
+  .getInitialCards()
+  .then((cards) => {
+    // cards is the list of cards that are on the server
+    section.renderItems(cards);
+  })
+  .catch((error) => {
+    console.log("error fetching cards", error);
+  });
 
-function renderCard(item, method = "addItem") {
-  const cardElement = createCard(item);
-  section[method](cardElement);
+function createCard(items) {
+  const cardElement = new Card(
+    items,
+    "#card-template",
+    handleImageClick,
+    handleDeleteCard,
+    handleLikeButton
+  );
+  return cardElement.getView();
 }
 
 // USER INFO -----
 
 const userInfo = new UserInfo({
-  profileNameSelector: ".profile__title",
-  profileDescriptionSelector: ".profile__description",
+  title: ".profile__title",
+  description: ".profile__description",
+  avatar: ".profile__image",
 });
 
-// POPUP WITH IMAGE -----
+api
+  .getUserInfo()
+  .then((data) => {
+    userInfo.setUserInfo({
+      title: data.name,
+      description: data.about,
+      avatar: data.avatar,
+    });
+  })
+  .catch(console.error);
+
+// LIKE AND UNLIKE -----
+
+function handleLikeButton(cardElement) {
+  if (!cardElement.isLiked) {
+    api
+      .likeCard(cardElement._id)
+      .then(() => {
+        cardElement.isLiked = true;
+        this._handleLikeIcon();
```

Comment:
`_handleLikeIcon`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.